src/test/resources/MOBS.xml

```diff
@@ -216,46 +216,52 @@
     <geo:siteLog gml:id="SITELOG_1">
         <geo:atSite xlink:href="#SITE_1"/>
         <geo:formInformation>
-            <geo:preparedBy/>
```

Comment:
The existing gml:ids in this document are in upper case. I think we should be consistent (at least within a file) and change the existing ids to lower case or add the new ones in upper case.

Reply:
Good catch. I have changed the three new gml:ids to upper case.